src/components/calcite-block/calcite-block.e2e.ts

```diff
@@ -19,6 +19,8 @@ describe("calcite-block", () => {
       }
     ]));
 
+  it("is accessible", async () => accessible(`<calcite-block heading="required"></calcite-block>`));
```

Comment:
If possible, doing one check with "everyone on" should suffice. But if there are multiple prop values that show different markup we'll have to do additional checks.  The heading the only thing we need for the block, or are there other props that should be used here?

Reply:
Looks like we should add the `open` property, and add some slotted content.

Reply:
Good idea. 

> Looks like we should add the open property, and add some slotted content.

☝️ should cover it since it will produce the most complete markup.